test/distributed/org/apache/cassandra/distributed/test/cql3/MultiNodeTableWalkBase.java

```diff
@@ -92,27 +92,6 @@ public MultiNodeState(RandomSource rs, Cluster cluster)
             super(rs, cluster);
         }
 
-        @Override
-        public boolean allowNonPartitionQuery()
```

Comment:
Caleb has worked hard to fix known issues, so we are finally ready to enable search expressions in multi node clusters again!